index.js

```diff
@@ -20,7 +28,16 @@ async function underPressure (fastify, opts) {
   var heapUsed = 0
   var rssBytes = 0
   var eventLoopDelay = 0
-  var lastCheck = now()
+  var lastCheck
+  var h
```

Comment:
What is h?

Reply:
`h` would be the `Histogram` instance returned by `monitorEventLoopDelay` (https://nodejs.org/dist/latest-v12.x/docs/api/perf_hooks.html#perf_hooks_class_histogram). Maybe I could call it `histogram`.

Reply:
call it histogram